java-sdk/connectors/gradle.properties

```diff
@@ -6,12 +6,12 @@ connectorsSDKVersion=1.1.0
 
 # Important!
 # change the fusion home
-# The folder where fusion was downloaded ie: /opt/fusion/4.0.1
+# The folder where fusion was downloaded ie: /opt/fusion/4.2.0
 # Note: it needs to include the path full path until the Fusion version
-fusionHome=/path/to/fusion/4.1.0
+fusionHome=/path/to/fusion/installation
 
 # Fusion target version, it should match the Fusion version from the fusion homme
 stopPort=55051
-fusionVersion=4.1.0
+fusionVersion=4.2.0-SNAPSHOT
```

Comment:
I think we should wait until 4.2.0 is release and change it to the actual release number.